modules/python/init.zsh

```diff
@@ -34,6 +34,51 @@ if (( ! $+commands[python] && ! $+commands[pyenv] )); then
   return 1
 fi
 
+function _python-workon-cwd {
+    if [[ -z "$WORKON_CWD" ]]; then
```

Comment:
Not that I'm aware of... the previous version worked because it wasn't a local variable and because there were other blocks other than this one.

I'd say remove it, and we can add it back later if needed... because this isn't enabled by default and it's a new feature, I'm fine with fixing this later (if it comes up at all)